modules/ROOT/pages/_partials/include-monitoring-metrics-collection.adoc

```diff
@@ -0,0 +1,6 @@
+= How Anypoint Monitoring Collects Metrics
+
+The Anypoint Monitoring agent collects metrics and aggregates them on a per-minute basis, so a minute is the lowest possible time increment you can use in dashboards. Metrics collection begins at a random second and not necessarily the top of the minute, and collection repeats at the same second in subsequent minutes. For example, if the agent collects metrics at 12:00:15, it collects them again at 12:01:15, 12:02:15, and so on.
+
+Within a Mule application, metrics associated with a connector (such as an HTTP Listener on a particular port, or an HTTP Outbound call to a particular endpoint) have their own individual starting seconds. For example, the inbound metrics for calls coming on port 8080 can be collected at 12:00:45, 12:01:45, 12:02:45, and so on. Meanwhile, metrics for outbound calls made to an endpoint can be collected at 12:00:23, 12:01:23, 12:02:23, and so on.
+For the purpose of aligning invocations between two Mule applications (such as when app A calls app B), the per-minute counts do not necessarily match precisely. If app A reports ten outbound calls made at 12:05, app B might report that the total number of corresponding inbound calls is split between 12:04 and 12:05, or 12:05 and 12:06 time windows (namely, 12:04 reports four calls and 12:05 reports six calls, totaling the ten calls). 
```

Comment:
_For the purpose of aligning_ > _To align_

Also, _invocation_ = ? _call times_? _metrics collection_?

Reply:
Is it counterintuitive to state the purpose of _alignment_, but then use _do not match precisely_? I don't understand how the mismatch between calls outbound for app A and inbound for app B causes alignment.

Reply:
Is this mismatch the result of the previous paragraph: the different reporting times?
So maybe what we want is to make that more clear.
I attempted to put that into words below...but I got stumped; I'm sure you can do better!

"Because inbound call metrics collection occurs at different seconds during a minute than call metrics for outbound calls, the total number of inbound and outbound calls reported during any minute might vary. 
For example, a metrics report might show that App A made 10 calls to App B at App A's collection point at 12:05, but App B might show that it collected ....ACK!

Reply:
Here's an idea: introduce App A and App B at the beginning, give example collection points, and then use that same scenario to illustrate how 10 calls from A for one (clock) minute might be 4 calls to B during that same clock minute, and 6 during the minute...after?

Reply:
An illustration would help a whole lot!